integration_tests/README.md

```diff
@@ -409,6 +409,13 @@ the SHS supported values for the config key
 With `zstd` it's easy to view / decompress event logs using the CLI `zstd -d [--stdout] <file>`
 even without the SHS webUI.
 
+### Worker Logs
+
+By default, integration tests will write the console output to a text file that 
+will appear under the run directories of the form
+`integration_tests/target/run_dir-<timestamp>-xxxx/gw0_worker_logs.log`. If xdist is not used (e.g., `TEST_PARALLEL=1`)
+there will be no logs generated besides the one on console.
```

Comment:
I think you should just say where the log is when test parallel =1 ... as far as I know we don't have that documented

Reply:
Also point to the log4j properties file and state how you might change the log level